QualifiedTeachersApi/src/QualifiedTeachersApi/Services/DqtReporting/DqtReportingService.cs

```diff
@@ -148,44 +147,20 @@ internal async Task ProcessChanges(CancellationToken cancellationToken)
                 },
                 async (entityType, ct) =>
                 {
-                    var stopwatch = Stopwatch.StartNew();
-                    int processedForEntityType = 0;
-
                     try
                     {
+                        int processedForEntityType = 0;
                         await ProcessChangesForEntityType(entityType, processedCount => processedForEntityType = processedCount, ct);
-                    }
-                    catch (Exception ex)
-                    {
-                        if (!(ex is OperationCanceledException && ct.IsCancellationRequested))
-                        {
-                            _telemetryClient.TrackException(new ExceptionTelemetry()
-                            {
-                                Exception = ex,
-                                Properties =
-                                {
-                                    { "Entity type", entityType }
-                                }
-                            });
-                        }
-
-                        throw;
-                    }
-                    finally
-                    {
-                        stopwatch.Stop();
-
-                        _telemetryClient.TrackMetric(new MetricTelemetry()
-                        {
-                            Name = $"{MetricPrefix}Batch processing time - {entityType}",
-                            Sum = stopwatch.Elapsed.TotalSeconds
-                        });
 
                         lock (metricsGate)
                         {
                             totalUpdates += processedForEntityType;
                         }
                     }
+                    catch (Exception ex)
+                    {
+                        throw new Exception($"Failed processing changes for '{entityType}' entity.", ex);
```

Comment:
Adding the entity type into the logged exception here so it's a little easier to debug issues.